### PR TITLE
add new + notable assets

### PR DESCRIPTION
**Pull request by jxue16**

### CODE REVIEW

File: assets/ai2.yaml [hunk 1]
```diff
@@ -255,3 +255,25 @@
...
+
+- type: dataset
+  name: MADLAD-400
+  organization: AI2
+  description: MADLAD-400 is a document-level multilingual dataset based on Common Crawl, covering 419 languages in total.
+  created_date: 2023-09-09
+  url: https://arxiv.org/abs/2309.04662
+  datasheet: https://huggingface.co/datasets/allenai/MADLAD-400
+  modality: text
+  size: 3T tokens
```
Comment (rishibommasani) [inline assets/ai2.yaml:267]: do we write out or say T (not consistent with CulutraX directly above)

File: assets/meta.yaml [hunk 2]
```diff
@@ -793,3 +793,27 @@
...
+- type: model
+  name: Llama 3
+  organization: Meta
+  description: Llama 3 is the third generation of Meta AI's open-source large language model. It comes with pretrained and instruction-fine-tuned language models with 8B and 70B parameters that can support a broad range of use cases.
+  created_date: 2024-04-18
+  url: https://llama.meta.com/llama3/
+  model_card: https://github.com/meta-llama/llama3/blob/main/MODEL_CARD.md
+  modality: text; text
+  analysis: The models were evaluated based on their performance on standard benchmarks and real-world scenarios. These evaluations were performed using a high-quality human evaluation set containing 1,800 prompts covering multiple use cases. The models also went through red-teaming for safety, where human experts and automated methods were used to generate adversarial prompts to test for problematic responses.
+  size: 70B parameters
```
Comment (rishibommasani) [inline assets/meta.yaml:806]: we should make sure to update this when llama 3 400b comes out

File: assets/reka.yaml [hunk 3]
```diff
@@ -21,3 +21,24 @@
...
+  analysis: Reka Core was evaluated against leading models such as OpenAIs GPT-4, Claude-3 Opus, and Gemini Ultra on a variety of tasks and metrics including multimodal and human evaluation conducted by a third party. It was found to be competitive or even surpassing these models.
+  size: unknown
+  dependencies: []
+  training_emissions: unknown
+  training_time: few months
+  training_hardware: thousands of GPUs
+  quality_control: ''
+  access: limited
+  license: unknown
+  intended_uses: Core can be used in e-commerce, social media, digital content and video games, healthcare, robotics, and other industries for tasks that require multimodal understanding, coding, complex reasoning, and more.
```
Comment (rishibommasani) [inline assets/reka.yaml:41]: add Reka (i.e. Reka  Core)

File: assets/stability.yaml [hunk 4]
```diff
@@ -263,3 +263,25 @@
...
+  training_emissions: unknown
+  training_time: unknown
+  training_hardware: unknown
+  quality_control: To protect creator copyrights, for audio uploads, Stability AI partners with Audible Magic to use their content recognition (ACR) technology to power real-time content matching and prevent copyright infringement. Opt-out requests were honored during the training phase.
+  access: open
+  license: unknown
+  intended_uses: It can be used to generate melodies, backing tracks, stems, and sound effects.
+  prohibited_uses: Uploading copyrighted material for transformation.
+  monitoring: Advanced content recognition is used to maintain compliance and prevent copyright infringement.
+  feedback: Submit a Support Request.
```
Comment (rishibommasani) [inline assets/stability.yaml:287]: add link

File: assets/tokyo.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  name: Aurora-M
+  organization: Tokyo Institute of Technology, MIT-IBM Watson Lab, Sapienza University of Rome
+  description: Aurora-M is a 15B parameter multilingual open-source model trained on English, Finnish, Hindi, Japanese, Vietnamese, and code. 
+  created_date: 2024-04-23
+  url: https://arxiv.org/pdf/2404.00399
+  model_card: none
+  modality: text; text
+  analysis: Evaluated on all language datasets compared to similarly sized SOTA models, with Aurora-M achieving strong performance in most.
+  size: 15B parameters
+  dependencies: [StarCoderPlus]
```
Comment (rishibommasani) [inline assets/tokyo.yaml:12]: seems surprising that they took a code-only model if the purpose is to deal with natural language?

Comment (rishibommasani) [reply assets/tokyo.yaml:12]: (i.e. check this is correct)

**Review by rishibommasani — CHANGES_REQUESTED** [review]

File: assets/tokyo.yaml (hunk 5 above)
Comment (jxue16) [reply assets/tokyo.yaml:12]: it is rather odd, but this is what the paper says: " a new 15B continually pretrained red-teamed multilingual LLM built on top of the StarCoderPlus model "

File: assets/meta.yaml (hunk 2 above)
Comment (jxue16) [reply assets/meta.yaml:806]: yup!

**Review by rishibommasani — APPROVED** [review]

**Pull request merged**